packages/lodestar/src/network/error.ts

```diff
@@ -0,0 +1,17 @@
+import {RpcErrorEnum} from "../constants";
+
+/**
+ * Error of network req/resp
+ */
+
+export class RpcError extends Error {
+  public status: RpcErrorEnum;
+  constructor(message: string, status?: RpcErrorEnum) {
```

Comment:
I think an `RpcError` will always require a status and only optionally include a message.
probably better to have the signature be `constructor(status: RpcErrorType, message?: string)`
(then we don't need the `withStatus` static method)